src/register.js

```diff
@@ -176,27 +178,49 @@ function configureProvider(conf = {}) {
   }
 }
 
-export default function register(conf = {}) {
+function configureNetwork(conf = {}){
   const provider = configureProvider(conf)
   const eth = new Eth(provider)
   const registryAddress = conf.registry || REGISTRY
   const DidReg = new EthContract(eth)(DidRegistryContract)
   const didReg = DidReg.at(registryAddress)
+  
+  return { eth, registryAddress, didReg}
+}
+
+function confNetworks(networksConf=[]){
+  let networks={}
+  for(let i=0;i<networksConf.length;i++){
+    const net=networksConf[i];
+    networks[net.name]=configureNetwork(net);
+  }
+  return networks;
+}
+
+export default function register(conf = {}) {
+  
   const logDecoder = abi.logDecoder(DidRegistryContract, false)
 
-  const lastChanged = async identity => {
-    const result = await didReg.changed(identity)
+  const networks={
+    'mainnet': configureNetwork(conf),
```

Comment:
This is rather confusing to configure mainnet like this using the `conf` object and then also configuring multiple networks using `conf.networks` later.  Why not include the mainnet config as part of the `networks.json` instead of maintaining a unique edge case?